src/commands/flow/ProjectPlanningRefactor.ts

```diff
@@ -0,0 +1,80 @@
+import {ICommand, ICommandParameters} from '../models';
+import * as simpleGit from 'simple-git';
+import * as path from 'path';
+import {PROJECT_PLANNING_PARAM} from './models';
+import {Repository} from '../../git';
+
+export class ProjectPlanningRefactor implements ICommand {
+    private static readonly PATH_TO_PROJECT_PLANNING_REPO: string = 'pathProjectPlanning';
+    private readonly CPLACE_TEMP_REPO: string = 'cplace-temp-repo';
+
+    private targetBranchName: string;
+    private cplaceRepo: simpleGit.Git;
```

Comment:
Type for `cplaceRepo` and `projectPlanningRepo` is wrong

Reply:
👍